### ISSUE TITLE
open position

### ISSUE THREAD

**Issue by Pandena** [Xypla/CSGORollBetBot#1]
Hello, are you still an active developer in cs? We would like to hire you to work on a p2p skin trading system for our platform

**Comment by Xypla** [comment]
Yeah i‘m still an active developer. Feel free to send me contacts.

**Comment by Pandena** [comment]
Please Contact @Arya_Elsher on Telegram, he describe the details and the task at hand
